core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -652,7 +646,11 @@ class HttpGoogleServicesDAO(
       }
 
       res <- allProcessedChunks.map(_.flatten).unsafeToFuture()
-    } yield res
+    } yield {
+        res collect {
+          case account if (firecloudHasAccess.isEmpty || account.firecloudHasAccess == firecloudHasAccess.getOrElse(true)) => account
```

Comment:
Default value in `firecloudHasAccess.getOrElse(true)` won't be used because of the earlier `isEmpty` check. Is there a clearer way to do the comparison?